RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETClass.cs

```diff
@@ -7,34 +7,44 @@ namespace RepositoryCompiler.CodeModel.CaDETModel.CodeItems
 {
     public class CaDETClass
     {
-        public string Name { get; set; }
-        public string FullName { get; set; }
-        public string SourceCode { get; set; }
-        public CaDETClass Parent { get; set; }
-        public List<CaDETModifier> Modifiers { get; set; }
-        public List<CaDETMember> Methods { get; set; }
-        public List<CaDETMember> Fields { get; set; }
-        public CaDETClassMetrics Metrics { get; set; }
+        public string Name { get; internal set; }
+        public string FullName { get; internal set; }
+        public string SourceCode { get; internal set; }
+        public CaDETClass Parent { get; internal set; }
+        public List<CaDETModifier> Modifiers { get; internal set; }
+        public List<CaDETMember> Members { get; internal set; }
```

Comment:
This is a much better name. I like it. 💯
There should be no confusion now.

Reply:
### Comment

Member is good for C# but, is it good for other languages ? For now this is good, and i agree but definitely we will need to refactor this to make it more suitable for other languages.

### Proposal

When we start to work on Java and others, make brainstorming about this and resolve this.

Reply:
Agreed. We also might decompose Members into accessors, properties and constructors in time.